src.fork5/Backend/Core/Engine/Base/Action.php

```diff
@@ -187,18 +187,18 @@ public function execute(): void
         $this->header->addJsData(
             'MediaLibrary',
             'browseAction',
-            BackendModel::createUrlForAction('MediaBrowser', 'MediaLibrary') . '#tabLibrary'
+            BackendModel::createUrlForAction('MediaBrowser', 'MediaLibrary') . '#tab-library'
```

Comment:
had Jelmer niet gezegd om niks aan te passen in src.fork5?

Reply:
fixed cdf93fcc1e00de9342ce8e526c67c452a2866496